content/en/docs/10/01/_index.md

```diff
@@ -4,9 +4,9 @@ weight: 11
 sectionnumber: 10.1
 ---
 
-Often you're going to use some kind of libraries, tools or dependencies during the build phase of your application that are not necessary during the execution time of the container. Since we want to keep the actual artifact as independent and small as possible, we often remove these dependencies in the docker build phase after the application itself is built.
+Often you're going to use some kind of libraries, tools or dependencies during the build phase of your application that are not necessary during the execution time of the container. We want to keep the actual artifact as independent and small as possible. So we often remove these dependencies in the docker build phase after the application itself is built.
 
-With docker 17.05 they implemented a solution for that problem, the so called multi stage-builds.
+Since docker version 17.05 there is a solution for that problem: the so called multi stage-builds.
```

Comment:
Looks like this was 5 years ago, I think we can safely remove the version information.

```suggestion
There is a solution to that problem: The so called multi-stage build.
```

Reply:
removed